com.unity.perception/Runtime/GroundTruth/Labelers/Visualization/Resources/SegRemoveBackgroundShader.shader

```diff
@@ -50,21 +50,26 @@
 
             fixed4 frag (v2f i) : SV_Target
             {
+                // Cap the max Transparency of the background so that we never
+                // reach 1.0 which causes an anomaly that the _SegmentTransparency
+                // value stops being honored.
+                if (_BackTransparency > 0.998f) _BackTransparency = 0.998f;
```

Comment:
I think the issue you're seeing is that the _BackTransparency of 1.0 is matching the alpha channel of the objects themselves (which in our labelers is always 1.0, but is not necessarily always such).

What if you change it to
```
if (abs(col.r - _RemoveColor.r) < .001f && abs(col.g - _RemoveColor.g) < .001f && ... )
   col.a = _BackTransparency;
else
   col.a = _SegmentTransparency;
```

Reply:
You are right. Incorporated your bug fix.